File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        start = 0
        end = len(numbers) - 1
        res = [-1, -1]
        while(start < end):
            sum = numbers[start] + numbers[end]
            if(sum == target):
                res = [start+1, end+1]
                return res
            elif sum < target:
                start += 1
            else:
                end -= 1
        return res

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int start = 0;
        int end = numbers.length - 1;
        int[] ans = {-1,-1};
        while(start < end){
            int sum = numbers[start] + numbers[end];
            if(sum < target){
                start++;
            }
            else if(sum > target){
                end--;
            }
            else{
                ans[0] = start+1;
                ans[1] = end+1;
                return ans;
            }
        }
        return ans;
    }
}